webapi/src/test/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala

```diff
@@ -751,10 +761,13 @@ class KnoraSipiIntegrationV1ITSpec
       // Upload the PDF file to Sipi.
       val pdfUploadResponse: SipiUploadResponse = uploadToSipi(
         loginToken = loginToken,
-        filesToUpload = Seq(FileToUpload(path = pathToMinimalPdf, mimeType = MediaTypes.`application/pdf`))
+        filesToUpload = Seq(
+          FileToUpload(path = pathToMinimalPdf, mimeType = org.apache.http.entity.ContentType.create("application/pdf"))
```

Comment:
why does this one need `ContentType.create()`?

Reply:
not all types are available as members inside the library.